OWNERS

```diff
@@ -4,27 +4,21 @@ reviewers:
 - bwplotka
 - simonpasquier
 - slashpai
-- prashbnair
 - arajkumar
 - sthaha
-- PhilipGough
-- bison
-- fpetkovski
 - raptorsun
 - jan--f
+- JoaoBraveCoding
 
 approvers:
 - bwplotka
 - simonpasquier
 - slashpai
-- prashbnair
 - arajkumar
 - sthaha
-- PhilipGough
-- bison
-- fpetkovski
```

Comment:
We should have prashbnair and fpetkovski as as emeritus approvers

Reply:
Done :)